docs/en/connector/sink/Jdbc.mdx

```diff
@@ -75,9 +75,9 @@ The transaction isolation level, which applies to current connection. The defaul
 ### customUpdateStmt [string]
 
 Configure when `saveMode` is specified as `update` , which is used to specify the update statement template for key conflicts
-
-Refer to the usage of `INSERT INTO table (...) values (...) ON DUPLICATE KEY UPDATE... ` of `mysql` , use placeholders or fixed values in `values`
-
+if `customUpdateStmt` is empty ,the sql will auto-generate for all columns, else use your sql which refer to the usage of
+`INSERT INTO table (...) values (...) ON DUPLICATE KEY UPDATE... ` of `mysql` , use placeholders or fixed values in `values`
+tips: the tableName of sql should be consistent with the dbName.
```

Comment:
What's the meaning of this? Do you mean that the table in sql statement should be consistent with `dbTable`? If so, please describe it more clearly

Reply:
![image](https://user-images.githubusercontent.com/35491928/190901422-ce94c0c8-899a-45e6-b0fd-13ed8a04b00f.png)

1. In many scenarios, do not need to customize the updateSQL, which can be generated automatically. In addition, user-defined sql is prone to exceptions that the number of parameters does not match.

2. If need customize SQL exactly, the table name of SQL should be consistent with that of dbTable

Reply:
Yep, but in your description. it's `dbName`

![image](https://user-images.githubusercontent.com/51053924/190902067-d8c40acd-bc6e-4710-9612-4f822a090f0e.png)

Reply:
My mistake,I fix it.